packages/react-admin/src/mui/list/List.js

```diff
@@ -268,7 +269,9 @@ export class List extends Component {
                         </div>
                     ) : (
                         <CardContent style={styles.noResults}>
-                            {translate('ra.navigation.no_results')}
+                            <Typography type="body1">
```

Comment:
this is a "cavalier" change (it has nothing to see with the rest of the PR) but it's not worth a PR on its own.